service/src/java/org/apache/hive/service/cli/HiveSQLException.java

```diff
@@ -150,4 +191,13 @@ public static TStatus toTStatus(Exception e) {
     return tStatus;
   }
 
+  @Override
+  public String getMessage(){
+    String errorMsg = super.getMessage() == null ? "Error" : super.getMessage();
+    if (!errorMsg.contains(ServiceContext.QUERY_ID)) {
```

Comment:
curious: is the contains to avoid appending queryId multiple times, there could be cases where query id is already appended & we want to avoid appending it again?
Something like that?